projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-write/query-write-respond-to-query/query-write-respond-to-query.component.html

```diff
@@ -19,20 +19,40 @@ <h1 class="govuk-heading-l govuk-!-margin-bottom-0">{{ 'Query details' | rpxTran
 
 <div class="query-respond">
   <div class="govuk-!-margin-bottom-4">
-    <h1 class="govuk-heading-m">{{ 'Respond to a query' | rpxTranslate }}</h1>
+    <h1 class="govuk-heading-m">
+      {{ context === 'Responded' ? 'Respond to a query' : 'Ask a follow-up question' | rpxTranslate }}
```

Comment:
@DavidJayakumar - similar to @MateiHMCTS's comment, please use the enum value instead of the string `'Responded'`